tf2onnx/onnx_opset/rnn.py

```diff
@@ -169,3 +168,84 @@ def replace_output(old_output, new_output):
     @classmethod
     def version_7(cls, ctx, node, **kwargs):
         cls.version_1(ctx, node, **kwargs)
+
+
+@tf_op("CudnnRNN")
+class CudnnRNN:
+    @classmethod
+    def version_11(cls, ctx, node, **kwargs):
```

Comment:
why opset 11?

Reply:
Need to rewrite function body to cater opset 10 ops - slice for example is the main one that has differed specification between 10 and 11. Shall we?

Reply:
Lowered to opset 10, current code works fine.